ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1077,3 +1077,197 @@
+                ip_version: 'ds'
+                jenkins_location: 'cloud'
+                kind_cluster_name: '{test_name}'
+          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-aws-ipv6-ds-(networkpolicy|all).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          status_context: jenkins-kind-ipv6-ds-networkpolicy
+          status_url: --none--
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-kind-aws-ipv6-ds-networkpolicy to re-trigger.
+          error_status: Failed. Add comment /test-kind-aws-ipv6-ds-networkpolicy to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
+          - timeout:
+              fail: true
+              timeout: 135
+              type: absolute
+          publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: kind-aws-ipv6-ds-conformance
+          node: 'antrea-kind-test'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+             - ${{sha1}}   
+          builders:
+            - builder-conformance-kind:
+                conformance_type: 'conformance'
+                ip_version: 'ds'
+                jenkins_location: 'cloud'
+                kind_cluster_name: '{test_name}'
+          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-aws-ipv6-ds-(conformance|all).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          status_context: jenkins-kind-ipv6-ds-conformance
+          status_url: --none--
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-kind-aws-ipv6-ds-conformance to re-trigger.
+          error_status: Failed. Add comment /test-kind-aws-ipv6-ds-conformance to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
+          - timeout:
+              fail: true
+              timeout: 135
+              type: absolute
+          publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: kind-aws-ipv6-ds-e2e
+          node: 'antrea-kind-test'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+             - ${{sha1}}      
+          builders:
+            - builder-e2e-kind:
+                 ip_version: 'ds'
+                 kind_cluster_name: '{test_name}'
+          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-aws-ipv6-ds-(e2e|all).*
```

Comment:
We can remove 'ds' in trigger phase for all ds jobs to align with the current implementation. You can refer to https://github.com/antrea-io/antrea/blob/e469bd9b67727e84120af30504850730b722d2e6/ci/jenkins/jobs/projects-lab.yaml#L127 .